frontend/catalyst/jax_primitives.py

```diff
@@ -77,7 +77,7 @@
 from catalyst.utils.extra_bindings import FromElementsOp, TensorExtractOp
 from catalyst.utils.types import convert_shaped_arrays_to_tensors
 
-# pylint: disable=unused-argument,abstract-method,too-many-lines
+# pylint: disable=unused-argument,too-many-lines
```

Comment:
```suggestion
# pylint: disable=unused-argument,abstract-method,too-many-lines
```
I think this is important unfortunately, have definitely seen that warning floating around. Super annoying that they don't appear to be very consistent :/